internal/actions/runWorkflow.go

```diff
@@ -72,7 +72,7 @@ func RunWorkflow() error {
 
 	success := false
 	defer func() {
-		if (!success || environment.GetMode() == environment.ModeDirect) && !environment.IsDebugMode() {
+		if (!success || environment.GetMode() == environment.ModeDirect) && !environment.IsDebugMode() && mode != environment.ModeTest {
```

Comment:
This condition is out of control, can we make it more readable?

Reply:
Ya. Will add a function